packages/babel-plugin-transform-function-name/test/fixtures/function-name/unicode-id-not-supported/options.json

```
@@ -0,0 +1,4 @@
+{
+    "plugins": ["transform-unicode-escapes"],
```

```suggestion
    "plugins": ["transform-function-name", "transform-shorthand-properties"],
```
See also https://github.com/babel/babel/pull/14047/files/673c623ec94a8daf6bce79b87b5f1f6c8f5bca3d..af6dcf7895b497080d54557977792fe7d0d26571#r770063967 and https://github.com/babel/babel/pull/14047#discussion_r770077450